src/typedefs.ts

```diff
@@ -0,0 +1,14 @@
+export enum ErrorType {
+  NONE = '',
+  LOAD = 'Unable to load todos',
+  ADD = 'Unable to add a todo',
+  UPDATE = 'Unable to update a todo',
+  DELETE = 'Unable to delete a todo',
+  TITLE = 'Title can\'t be empty',
```

Comment:
To avoid characters escaping it's better to use template literals
```suggestion
  TITLE = `Title can't be empty`,
```

Reply:
Fixed

Reply:
Fixed